robot-name/robot-name_test.spec.js

```diff
@@ -12,9 +12,17 @@ describe("Robot", function() {
   });
 
   it("different robots have different names", function() {
-    var robotOne = new Robot();
-    var robotTwo = new Robot();
-    expect(robotOne.name).not.toEqual(robotTwo.name);
+    var usedNames= {};
+
+    var i = 0,
+        max = 10000;
+
+    for (i; i < max; i++) {
```

Comment:
I'm not a JS dev, either. While I also like for (var i = 0; i < 1000; i++)
better, I'll defer to rchavarria.

Zachary Smith


On Thu, Aug 13, 2015 at 9:02 PM, Katrina Owen <notifications@github.com>
wrote:

> In robot-name/robot-name_test.spec.js
> <https://github.com/exercism/xjavascript/pull/96#discussion_r37042330>:
>
> > @@ -12,9 +12,17 @@ describe("Robot", function() {
> >    });
> >
> >    it("different robots have different names", function() {
> > -    var robotOne = new Robot();
> > -    var robotTwo = new Robot();
> > -    expect(robotOne.name).not.toEqual(robotTwo.name);
> > +    var usedNames= {};
> > +
> > +    var i = 0,
> > +        max = 10000;
> > +
> > +    for (i; i < max; i++) {
>
> Wow, ok that does make sense. I still like it better with for (var i = 0;
> i < 1000; i++), to be honest.
>
> I'm not a JavaScript dev, though, so if the two of you can agree on
> something, I'll leave the final decision to you :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/exercism/xjavascript/pull/96/files#r37042330>.
>